inst/unitTests/cpp/misc.cpp

```diff
@@ -179,22 +179,22 @@ void test_stop_variadic() {
 }
 
 // [[Rcpp::export]]
-bool testNullableForNull(Nullable<NumericMatrix> M) {
+bool testNullableForNull(const Nullable<NumericMatrix>& M) {
```

Comment:
Methinks we may want these as `const` variants (post PR) as well as non-`const` variants just to be sure.